lib/src/model/movie_result.dart

```diff
@@ -1,36 +1,25 @@
 import '../utils/movie_string.dart';
 
 class MoviesResult {
-  int? voteCount;
   late int id;
-  bool? video;
-  double? voteAverage;
-  String? title;
-  double? popularity;
-  String? posterPath;
-  String? originalLanguage;
+  late num voteAverage;
+  late String title;
+  late String posterPath;
   String? originalTitle;
   List<int> genreIDs;
-  String? backdropPath;
-  bool? adult;
-  String? overview;
-  String? releaseDate;
+  late bool adult;
```

Comment:
Whats the purpose of these variables being late instead of nullables ?

Reply:
I took a look again and there was no point with this variables being late. I rebuilt this part.